README.md

```diff
@@ -15,6 +15,9 @@ $ ./gradlew install
 This builds and installs the program into `build/install/gradle-enterprise-api-samples`.
 You can use the `build/install/gradle-enterprise-api-samples/bin/gradle-enterprise-api-samples` script to run the sample.
 
+### Note on JDK11
+
+Due to a change in the [openapi generator project](https://github.com/gradle/gradle-enterprise-api-samples/pull/74)  JDK 11 is required to build the samples. The clients still work using java 8
```

Comment:
Missing comma before 'JDK 11'.

Reply:
Shouldn't that be 'JDK 8'?   Also, that sentence is missing a period.

Reply:
No it's not JDK 8, because running an app does not need a JDK but a JRE, but using JRE would've been kindof weird IMO

Reply:
We should capitalize 'Java'.